bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/PredefinedTypes.java

```diff
@@ -175,10 +176,10 @@ public class PredefinedTypes {
     public static final MapType TYPE_DETAIL;
     public static final Type TYPE_ERROR_DETAIL;
     public static final ErrorType TYPE_ERROR;
-    public static final BUnionType TYPE_CLONEABLE;
+    public static final UnionType TYPE_CLONEABLE;
 
-    public static final BUnionType TYPE_JSON_DECIMAL;
-    public static final BUnionType TYPE_JSON_FLOAT;
+    public static final UnionType TYPE_JSON_DECIMAL;
+    public static final UnionType TYPE_JSON_FLOAT;
 
```

Comment:
Do we need private constructor here?

Reply:
Moved it up with d916107c901b00ea0d5284469eaa0cfb4b2948a2.